geronimo-json_1.0_spec/src/main/java/javax/json/spi/JsonProvider.java

```diff
@@ -28,6 +28,9 @@
 import javax.json.stream.JsonGeneratorFactory;
 import javax.json.stream.JsonParser;
 import javax.json.stream.JsonParserFactory;
+
+import org.apache.geronimo.osgi.locator.ProviderLocator;
```

Comment:
do you care updating it to do it by reflection? I'd like to keep that dependency running without osgi-locator if possible